build.gradle

```diff
@@ -86,6 +87,8 @@ allprojects {
         }
         testCompile "org.mockito:mockito-core:$mockitoVersion"
         testCompile group: "org.mock-server", name: "mockserver-netty", version: "$mockServerVersion"
+        testCompile group: "org.powermock", name: "powermock", version: "$powermockVersion"
+        testCompile group: "org.powermock", name: "powermock-api-mockito-common", version: "$powermockVersion"
```

Comment:
These two should be replace with (I am using short hand notation for the dependencies) :
```
testCompile "org.powermock:powermock:$powermockVersion"
testCompile "org.powermock:powermock-api-mockito:$powermockVersion"
testCompile "org.powermock:powermock-module-junit4:$powermockVersion"
```

Based on http://stackoverflow.com/questions/18645465/mockito-and-powermock-dependency-issues - suggested by Mockito Maven to include the `module-junit4`.